utils/data_generator.py

```diff
@@ -56,7 +68,7 @@ def __data_load(self, imgs, labels, start_pos: int, from_i: int, to_i: int):
                 file_content = source_data.get_content(cache=USE_CACHE)
                 file_like_object = io.BytesIO(file_content)
                 img = load_img(file_like_object, target_size=(IMG_ROWS, IMG_COLS))
-                img = preprocessor(img)
+                img = preprocessor.transform(img, is_train=self.is_train, seed=RANDOM_SEED)
```

Comment:
seems better to do this after if-else clause because it is common in each condition.

Reply:
Thank you!